platforms/configs/BUILD

```diff
@@ -0,0 +1,49 @@
+package(default_visibility = ["//visibility:public"])
+
+config_setting(
+    name = "linux_x86_64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+config_setting(
+    name = "macos_x86_64",
+    constraint_values = [
+        "@platforms//os:macos",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+config_setting(
+    name = "macos_arm64",
+    constraint_values = [
+        "@platforms//os:macos",
+        "@platforms//cpu:arm64",
+    ],
+)
+
+constraint_setting(name = "docker_daemon")
```

Comment:
nit: I would prefer to call this `dockerd` mostly because it's shorter but also because it's the official name: [doc](https://docs.docker.com/engine/reference/commandline/dockerd/)

Feel free to ignore if you think it'll cause confusion though :sweat_smile: